# TODO bessere bezeichnungen

queue_max_size: 10

# TODO publish debug_image on/off

threshold: 0.6
kernel_size: 3